src/components/VolunteerEventsList/index.tsx

```diff
@@ -11,23 +11,66 @@ import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
 import Paper from "@material-ui/core/Paper";
 import SortIcon from "@material-ui/icons/Sort";
-import Pagination from "@material-ui/lab/Pagination";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
+import ChevronRightIcon from "@material-ui/icons/ChevronRight";
+import urls from "utils/urls";
 
-// create attended events table for current volunteer
-export default function VolunteerEventsList(props: Volunteer) {
+const VolunteerEventsList = (vol: Volunteer) => {
     const classes = useStyles();
-    const eventsPerPage = 5;
-    const [page, setPage] = useState(1);
+    const eventsPerPage = 3;
```

Comment:
Typically frontend shouldn't know about page size, but since we're low on time we can keep this here👌 . We just have to remember to change page size in 2 places.